lib/open_project/translations/helpers/tmp_directory.rb

```diff
@@ -0,0 +1,14 @@
+module TmpDirectory
+  def create_tmp_directory(delete: nil)
+    tmp_path = File.join(FileUtils.pwd, 'tmp')
+    if delete
+      FileUtils.rm_rf tmp_path
+    end
+    FileUtils.mkdir_p tmp_path
+    tmp_path
```

Comment:
I don't get yet why you would want to delete the tmp directory in this method. Maybe it is just a wrong layer of abstraction and you should have the methods:

`remove_tmp_directory` -> removes the directory
`create_tmp_directory` -> creates the directory (without the delete param)
 
and something like `tmp_directory` which will always give you a tmp directory regardless of whether it exists.

This comment might be irrelevant once I have read on.